modules/statics/src/index.ts

```diff
@@ -9,6 +9,8 @@ export {
   CeloCoin,
   ContractAddressDefinedToken,
   Erc20Coin,
+  Erc721Coin,
```

Comment:
I think we can revert changes in `modules/statics/src/index.ts` since we have taken a different approach